app/README/alert_actions.conf.spec

```diff
@@ -1,8 +1,7 @@
-
-[notebook]
-python.version          = python3
-param._cam              = <json> Active response parameters.
-param.notebook          = <string> Notebook
-param.paramOne      = <string> Field Name for Parameter One
-param.paramTwo      = <string> Field Name for Parameter Two
-
+[launch_notebook]
```

Comment:
it's not allowed to change command/parameter names, as people depend on it

Reply:
This is a breaking change but we can notify users in the change log.